pretrain/redpajama.py

```diff
@@ -131,19 +131,31 @@ def main(fabric: L.Fabric, train_data_dir: Path, val_data_dir: Optional[Path]) -
     )
     model, optimizer = fabric.setup(model, optimizer)
 
+    state = {
```

Comment:
Just fyi, i'll look into adding state_dict for the speed monitor in a follow-up

Reply:
Why does it need one?